rootfs/etc/nginx/lua/test/balancer/sticky_test.lua

```diff
@@ -263,3 +263,209 @@ describe("Sticky", function()
     end)
   end)
 end)
+
+describe("StickyWildcard", function()
```

Comment:
I'd have expected only single new test that would have cookie with wildcard set, and then assert that it would work for an FQDN matching the wildcard. Can you explain why are you adding all these tests?

Reply:
I wasn't sure on a good way to split out the test independently, but you're comment does clear it up for me.